synapse/tests/test_lib_scope.py

```diff
@@ -67,7 +68,7 @@ def test_lib_scope_enter(self):
         self.none(scope.get('no'))
         self.raises(IndexError, scope.leave)
 
-    def test_lib_scope_get_defval(self):
+    async def test_lib_scope_get_defval(self):
```

Comment:
This uses a local scope object. Why is it async?

Reply:
same same :)